package-lock.json

```diff
@@ -1267,9 +1267,9 @@
       "integrity": "sha512-kBJtf7PH6aWwZ6fka3zQ0p6SBYzx4fl1LoZXE2RrnYST9Xljm7WfKJrU4g/Xr3Beg72MLrp1AWNUmuYJTL7Cow=="
```

Comment:
so all changes are intentionally checked in?

Reply:
same as previous comment